src/core/channel/context.h

```diff
@@ -41,4 +41,9 @@ typedef enum {
   GRPC_CONTEXT_COUNT
 } grpc_context_index;
 
-#endif
+typedef struct {
+  void *value;
+  void (*destroy)(void *);
+} grpc_call_context;
```

Comment:
let's make this grpc_call_context_{value, element, xxx}... context is so overused, and it's not the 'whole context'